notebooks/vertex_genai/labs/retrieval_augmented_generation.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<ul><li>Add a markdown cell with s&nbsp;<code>### Exercise</code>&nbsp;title, and a description of the exercise.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>